_posts/2017-10-10-performance-left-right-and-center.md

```diff
@@ -0,0 +1,49 @@
+---
+layout:     post
+title:      Performance Left Right and Center.
+date:       2017-10-10
+summary:    Why we need a holistic approach to performance testing in Agile development  - it's not all about Shifting Left.
+image:      perf-lrc-frontpage.jpg
+category:   Testing
+tags:       Performance
+author:     paul_whitehead
+---
+
+### Performance Matters
+At Sky Betting & Gaming (SBG) we are continually looking to improve the service we provide to our customers. As an online business, a key part of this is user experience and in particular performance. There have been many studies over the years that show the close relationship between page load time and abandonment rate. Slow page load times and slow response times in general can be caused by a myriad of influencing factors such as slow network, inefficient code and over-utilised servers - each of which has to potential to cause customers to become frustrated, to miss out on placing an important bet, or ultimately turn away completely and move on to one our competitors. Left untended, theses issue could escalate into intermittent loss of service or even a full blown site outage. For a business like ours, this could be disastrous, both in terms of reputation and financially, no matter when it were to occur but particularly during a big sporting event like the Grand National.
+
+While the causes of performance problems can be many and varied, so too must the methods required to identify them. Effective risk mitigation requires an approach to performance testing that allows issues to be identified at the earliest opportunity, while also having a means to effectively assess production performance. At SBG we embrace agile ways of working like devops and continuous delivery allowing us to move faster by reducing delivery time and improve the quality on each release. Gone are the days when performance testing was the last step before deploying production with the potential to derail an important launch. In this fast paced environment, traditional gated waterfall methods need to make way for techniques that seek to accelerating the feedback loop.
```

Comment:
Really minor, devops seems to typically be DevOps. Googling devops always seems to provide links to it being written as DevOps anyway.